custom-menus/fzf_current_session_history_menu.nu

```diff
@@ -0,0 +1,36 @@
+# The part below should be pasted inside the 'menus' list of the 'config.nu' file 
+
+	{
+      # session menu
+      name: fzf_history_menu_fzf_ui_session
```

Comment:
since this menu doesn't use fzf, how about calling it history_menu_by_session or something like that? I think seeing fzf here may confuse people if it's not used.

Reply:
I had no idea what the 'fzf' mean:) So I just left this part from the original menus name. I renamed the menu names.

Reply:
fzf is a cool utility. you should look it up. thanks!